### DISCUSSION TITLE
Discussion about the world links

### DISCUSSION THREAD

**Discussion by ghost** [Yellow-Dog-Man/Resonite-Issues#2563] [Bugs & Issues]
This might not be the right way to go about this but I've been advised not to message the devs about this.

What I'm wondering is what the roadmap for getting the world links looks like and what blockers there are to getting this fixed as I feel it could be a downside to new and old users alike who might want to host sessions/meetups.

**Comment by LexiBasilisk** [comment]
Is this in relation to #1295, @ExoTheWicker?

The issue is on our tracker, and is known. It just hasn't been prioritized.

There are a number of factors that determine what we work on- this is all covered through our "[how we prioritize](https://github.com/Yellow-Dog-Man/Resonite-Issues/blob/main/HOW_WE_PRIORITIZE.md)" document.

If users wish to signal interest in it, there are ways they can do so such as [reacting to the initial issue with a thumb up](https://github.com/Yellow-Dog-Man/Resonite-Issues/blob/main/HOW_TO_UPVOTE.md).
